pkg/loop/internal/pb/relayer.proto

```diff
@@ -65,38 +64,29 @@ message RelayArgs {
 message PluginArgs {
   string transmitterID = 1;
   bytes pluginConfig = 2;
+  string providerType = 3;
 }
 
-// NewConfigProviderRequest has arguments for [github.com/smartcontractkit/chainlink-relay/pkg/loop.Relayer.NewConfigProvider].
-message NewConfigProviderRequest {
-  RelayArgs relayArgs = 1;
-}
-
-// NewConfigProviderReply has return arguments for [github.com/smartcontractkit/chainlink-relay/pkg/loop.Relayer.NewConfigProvider].
-message NewConfigProviderReply {
-  uint32 configProviderID = 1;
-}
-
-// NewMedianProviderRequest has arguments for [github.com/smartcontractkit/chainlink-relay/pkg/loop.Relayer.NewMedianProvider].
-message NewMedianProviderRequest {
+// NewPluginProviderRequest has arguments for [github.com/smartcontractkit/chainlink-relay/pkg/loop.Relayer.NewPluginProvider].
+message NewPluginProviderRequest {
   RelayArgs relayArgs = 1;
   PluginArgs pluginArgs = 2;
 }
 
-// NewMedianProviderReply has return arguments for [github.com/smartcontractkit/chainlink-relay/pkg/loop.Relayer.NewMedianProvider].
-message NewMedianProviderReply {
-  uint32 medianProviderID = 1;
+// NewPluginProviderReply has return arguments for [github.com/smartcontractkit/chainlink-relay/pkg/loop.Relayer.NewPluginProvider].
+message NewPluginProviderReply {
+  uint32 pluginProviderID = 1;
+  string providerType = 2;
```

Comment:
Is a type parameter necessary? I had imagined that the type would be implicit :thinking:

Reply:
I think it would be OK to omit this if it's always going to be the same as `PluginArgs.providerType`.